### ISSUE TITLE
GenericContainer withFileSystemBind: creates a root owned directory/file

### ISSUE THREAD

**Issue by nicolasduminil** [testcontainers/testcontainers-java#8871]
### Module

Core

### Testcontainers version

1.19.8

### Using the latest Testcontainers version?

Yes

### Host OS

Ubuntu 22.04.4 LTS

### Host Arch

x86

### Docker version

```shell
26.1.4
```

### What happened?

Using the following statement:

    new GenericContainer<>("...")
      ...
      .withFileSystemBind("...", "...", BindMode.READ_WRITE)
      ...;

creates a `root:root` directory/file while executed by a normal, non-root user. 

Is there any way to avoid that and to create directories/files owned by the current user ?

**Comment by bedla** [comment]
You have to specify host user to have non-root owner of the files.

I have create test example with description here https://github.com/bedla/testcontainers-created-host-file-owner .

TLDR; Just use `GenericContainer.withCreateContainerCmdModifier({ cmd -> cmd.withUser("1000:1000") })` and you will be ok. See https://github.com/bedla/testcontainers-created-host-file-owner/blob/master/src/test/kotlin/cz/publicstaticvoidmain/reproduce/tccreatedhostfileowner/MyApplicationTest.kt file for TC details.
